tests/cmdline/cmd_showbc.py.exp

```diff
@@ -47,13 +47,13 @@ arg names:
 42 IMPORT_STAR
 43 LOAD_CONST_NONE
 44 RETURN_VALUE
-File cmdline/cmd_showbc.py, code block 'f' (descriptor: \.\+, bytecode @\.\+ 45\[46\] bytes)
-Raw bytecode (code_info_size=8\[46\], bytecode_size=370):
- a8 12 9\[bf\] 03 05 60 60 26 22 24 64 22 24 25 25 24
+File cmdline/cmd_showbc.py, code block 'f' (descriptor: \.\+, bytecode @\.\+ 466 bytes)
+Raw bytecode (code_info_size=84, bytecode_size=382):
```

Comment:
According to the failing CI, we still need the regex here for the 2-byte variance in size. Why do we get different results when running on CI vs. locally?

Reply:
SETTRACE adds a few more line-number to bytecode mapping bytes, that's the reason.

It would be good to have equivalent bytecode for all configurations (ie with and without SETTRACE enabled).  Not sure if that's possible without increasing size.